spyder/app/tour.py

```diff
@@ -1255,6 +1255,76 @@ def any_has_focus(self):
              self.tips.hasFocus() or self.canvas.hasFocus())
         return f
 
+
+class OpenTourDialog(QDialog):
+    """Initial Widget with tour"""
+    sig_launch_tour_button_clicked = Signal()
+    # Signal triggered for the 'Dismiss' button
+    sig_dismiss_button_clicked = Signal()
+
+    def __init__(self, parent, tour_function):
+        super().__init__(parent)
+        self.setFixedHeight(170)
+        self.setFixedWidth(350)
+        self.tour_function = tour_function
+
+        # Image
+        images_layout = QHBoxLayout()
+        if is_dark_interface():
+            icon_filename = 'tour-spyder-logo.png'
+        else:
+            icon_filename = 'tour-spyder-logo.png'

```

Comment:
```suggestion
            icon_filename = 'tour-spyder-logo.png'
```

Since we're not doing anything different for dark vs light mode, this can be simplified.